vmicore/CMakeLists.txt

```diff
@@ -64,26 +78,40 @@ FetchContent_MakeAvailable(googletest)
 
 # Setup libvmi
 
-FetchContent_Declare(
-        libvmi
-        GIT_REPOSITORY https://github.com/GDATASoftwareAG/libvmi
-        GIT_TAG test
-)
-set(libvmi_ENABLE_STATIC OFF)
-set(libvmi_BUILD_EXAMPLES OFF)
+set(ENABLE_STATIC OFF CACHE INTERNAL "")
```

Comment:
Those are configurable options and it would therefore be confusing if they were not displayed by `ccmake`.